crates/storage/derive/src/tests/packed_layout.rs

```diff
@@ -22,20 +22,20 @@ fn unit_struct_works() {
         }
         expands to {
             const _: () = {
-                impl ::ink_storage::traits::PackedLayout for UnitStruct {
-                    fn pull_packed(&mut self, __key: &::ink_primitives::Key) {
+                impl ::ink::storage::traits::PackedLayout for UnitStruct {
```

Comment:
These tests will need to be moved to the `ink` crate if they are to work. Alternatively if these traits/derives are removed entirely by #1217 then it will no longer be an issue,